pyproject.toml

```
@@ -22,6 +22,7 @@ requests = "^2.32.0"
 urllib3 = "^2.2.1"
 pydantic = "^2.8.2"
 pyyaml = "^6.0.1"
+atlassian-python-api = "^3.41.14"
```

You can bump it to the new latest version:

```suggestion
atlassian-python-api = "^3.41.16"
```